<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>238</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>286</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>342</width>
    <height>300</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>397nm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Laser 2</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Laser 6</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Laser 1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Laser 3</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Laser 4</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Laser 5</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Laser 7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLCDNumber" name="lcdNumber"/>
   </item>
   <item row="3" column="1">
    <widget class="QLCDNumber" name="lcdNumber_2"/>
   </item>
   <item row="4" column="1">
    <widget class="QLCDNumber" name="lcdNumber_3"/>
   </item>
   <item row="5" column="1">
    <widget class="QLCDNumber" name="lcdNumber_4"/>
   </item>
   <item row="6" column="1">
    <widget class="QLCDNumber" name="lcdNumber_5"/>
   </item>
   <item row="7" column="1">
    <widget class="QLCDNumber" name="lcdNumber_6"/>
   </item>
   <item row="8" column="1">
    <widget class="QLCDNumber" name="lcdNumber_7"/>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QCheckBox" name="checkBox_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QCheckBox" name="checkBox_8">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
